transport/http/client.go

```diff
@@ -75,11 +74,11 @@ func (c Client) Endpoint() endpoint.Endpoint {
 
 		req, err := http.NewRequest(c.method, c.tgt.String(), nil)
 		if err != nil {
-			return nil, fmt.Errorf("NewRequest: %v", err)
+			return nil, Err{"NewRequest", err}
```

Comment:
Can the domains be exported constants in the package scope, so consumers may test for them?